KubeArmor/eventAuditor/entryPointHandler.go

```diff
@@ -16,48 +17,83 @@ import (
 // InitializeEntryPoints Function
 func (ea *EventAuditor) InitializeEntryPoints() bool {
 	// if something wrong, return false
+	b, err := bpf.OpenObjectFromFile("entrypoint.bpf.o")
+	must(err)
+	defer b.Close()
+
+	err = b.Load()
+	must(err)
 
 	return true
 }
 
-// DestoryEntryPoints Function
+/* DestoryEntryPoints Function
```

Comment:
why is this function empty? don't we need to clean up entrypoints when kubearmor is terminated?

Reply:
The entrypoint object is already being closed using Close() in InitializeEntryPoints() and we are detaching the entrypoint using DetachEntryPoint().  What do we actually want to do using this function?

Reply:
I don't get it.

I think this is how the entrypoint handler works.
When the InitializeEntryPoints() is terminated, the loaded object is closed.
Whenever we need to attach and detach some entrypoints, we find the object using b.FindProgramByName().

Then, don't we need to detach all entrypoints when KubeArmor is terminated?

Reply:
Yes, we are using DetachEntryPoint() function to detach entrypoints

Reply:
then, who is the caller of DetachEntryPoint() when KubeArmor is terminated?

Reply:
UpdateEntryPoints() will call both AttachEntryPoint() and DetachEntryPoint()